controller/src/beerocks/master/son_master_thread.cpp

```diff
@@ -621,7 +621,7 @@ bool master_thread::autoconfig_wsc_add_m2(std::shared_ptr<ieee1905_1::tlvWscM1>
     // Create ConfigData
     uint8_t buf[1024];
     WSC::cConfigData config_data(buf, sizeof(buf), false, true);
-    if (bss_info_conf) {
+    if (bss_info_conf && !bss_info_conf->ssid.empty() && !bss_info_conf->network_key.empty()) {
```

Comment:
I don't like this. I think this empty SSID is an artifact of the CAPI and not generic enough to warrant this special treatment here. So I think the empty SSID should be handled in handle_dev_set_config() instead.

Also, an empty network key definitely doesn't mean that it should be cleared. If it's an open network, there will be no network key.